pmd-dist/src/main/resources/scripts/run.sh

```diff
@@ -82,6 +82,29 @@ check_lib_dir() {
   fi
 }
 
+set_conf_dir() {
+  if [ -z ${CONF_DIR} ]; then
+    # Allow for symlinks to this script
+    if [ -L $0 ]; then
+      local script_real_loc=$(readlink "$0")
+    else
+      local script_real_loc=$0
+    fi
+    local script_dir=$(dirname "${script_real_loc}")
+    local cwd="${PWD}"
+
+    cd "${script_dir}/../conf"
+    readonly CONF_DIR=$(pwd -P)
+    cd "${cwd}"
```

Comment:
I think you can use pushd here, since we run with bash:
```suggestion
    pushd "${script_dir}/../conf"
    readonly CONF_DIR=$(pwd -P)
    popd
```